src/status_im/contexts/wallet/wallet_connect/responding_events.cljs

```diff
@@ -103,28 +103,28 @@
 (rf/reg-event-fx
  :wallet-connect/send-response
  (fn [{:keys [db]} [{:keys [result error]}]]
-   (let [{:keys [id topic] :as event} (get-in db [:wallet-connect/current-request :event])
-         method                       (wallet-connect-core/get-request-method event)
-         screen                       (wallet-connect-core/method-to-screen method)
-         web3-wallet                  (get db :wallet-connect/web3-wallet)]
-     {:fx [[:effects.wallet-connect/respond-session-request
-            {:web3-wallet web3-wallet
-             :topic       topic
-             :id          id
-             :result      result
-             :error       error
-             :on-error    (fn [error]
-                            (log/error "Failed to send Wallet Connect response"
-                                       {:error                error
-                                        :method               method
-                                        :event                :wallet-connect/send-response
-                                        :wallet-connect-event event})
-                            (rf/dispatch [:dismiss-modal screen])
-                            (rf/dispatch [:wallet-connect/reset-current-request]))
-             :on-success  (fn []
-                            (log/info "Successfully sent Wallet Connect response to dApp")
-                            (rf/dispatch [:dismiss-modal screen])
-                            (rf/dispatch [:wallet-connect/reset-current-request]))}]]})))
+   (when-let [{:keys [id topic] :as event} (get-in db [:wallet-connect/current-request :event])]
```

Comment:
This check is needed because we call this handler when modal is being dismissed => there might be a situation when the request was accepted already.